src/main/java/uk/gov/hmcts/juror/api/bureau/service/UserServiceImpl.java

```diff
@@ -317,7 +330,11 @@ public void assignUrgentResponse(final DigitalResponse urgentJurorResponse) thro
             updateResponse.setStaffAssignmentDate(now.toLocalDate());
 
             log.trace("Assigning juror response {}", urgentJurorResponse);
-            jurorResponseRepository.save(updateResponse);
+            if (updateResponse.getReplyType().getType().equals(ReplyMethod.DIGITAL.getDescription())) {
+                jurorResponseRepository.save((DigitalResponse) updateResponse);
+            } else {
+                jurorPaperResponseRepository.save((PaperResponse) updateResponse);
+            }
```

Comment:
Could this be pulled out into its own method?

(As it the same logic is used multiple times)